charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,63 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/ash
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/api/extensions/dam/fs-event"
+    while IFS= read -r line; do
+      status=$(echo "$line" | cut -d';' -f1)
```

Comment:
$mount variable needs to be added, including the .Values.webdav.events.mountPrefix value to relativize $path and $dest.

Reply:
Done